scripts/core/superdesk-api.d.ts

```diff
@@ -1413,6 +1413,11 @@ declare module 'superdesk-api' {
         getMediaId(attachment: IAttachment): IMedia['_id'];
     }
 
+    export interface IAssetTag {
```

Comment:
With your latest changes, `IAssetTag` is no longer required in core

Reply:
updated.